library/core/src/unicode/mod.rs

```diff
@@ -1,7 +1,9 @@
-#![unstable(feature = "unicode_internals", issue = "none")]
+#![stable(feature = "unicode_version", since = "1.45.0")]
```

Comment:
Necessary because `UNICODE_VERSION` and therefore `core::unicode::UNICODE_VERSION` is already stable.